runners/spark/src/main/java/org/apache/beam/runners/spark/stateful/SparkGroupAlsoByWindowViaWindowSet.java

```diff
@@ -204,6 +207,32 @@ public void outputWindowedValue(
         this.droppedDueToLateness = droppedDueToLateness;
       }
 
+      /**
```

Comment:
This change is unrelated to this PR.

Did you rebase on the latest `master` branch?